accounts/serializers.py

```diff
@@ -2,10 +2,36 @@
 from django.utils.crypto import get_random_string
 from rest_framework import serializers
 
-from accounts.models import Email, PhoneNumberModel, Student, User
+from accounts.mixins import ManyToManySaveMixin
+from accounts.models import Email, Major, PhoneNumberModel, School, Student, User
 from accounts.verification import sendEmailVerification, sendSMSVerification
 
 
+class SchoolSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = School
+        fields = ("id", "name")
+        extra_kwargs = {"id": {"read_only": False}}
```

Comment:
> What's the reasoning behind not having ID be read-only? That should be the case since no one can modify schools/majors

We were having an issue with being able to update a major or school with just the id instead of having to put id, name, and degree type. Looking up fixes yielded this solution becuase this enables us to "write" majors to a student using the id. Let me know if this is okay or if there's another/better solution @ArmaanT